tests/SecurityTest.php

```diff
@@ -5,35 +5,18 @@
 
 class SecurityTest extends \PHPUnit\Framework\TestCase
 {
-    public function depGetHashValue()
-    {
-        return Hash::create('bow');
-    }
-
-    public function depGetEncryptValue()
-    {
-        Crypto::setKey(file_get_contents(__DIR__.'/config/.key'), 'AES-256-CBC');
-
-        return Crypto::encrypt('bow');
-    }
```

Comment:
These above of two methods are not necessary.

I rewrite the following two tests and make their tests worked perfectly.